sources/README.md

```diff
@@ -1857,6 +1857,20 @@ _The Quattroshapes point gazetteer. A big list of point locations that supplemen
   	* **Global**: [GeoNames](http://www.geonames.org/) - The GeoNames geographical database covers all countries and contains over eleven million placenames that are available for download free of charge.
 * usage: `concordance`, `property`, `geometry`
 
+## Romanian National Agency for Cadastre and Land Registration (ANCPI)
+
+_The National Agency for Cadastre and Real Estate Advertising (ANCPI) is the sole authority in the field of cartography, cadastre and real estate advertising in Romania._ 
+
+* added: `2020-04-17`
+* id: `1713095439`
+* name: `ro-ancpi`
+* prefix: `ro-ancpi`
+* license_type: _License for Open Government - OGL ROU 1.0_
+* license_text: _You are allowed to: Copy, publish, distribute the information, To adapt the information, Re-use the information for commercial or non-commercial purposes. Respecting the following conditions: you must declare the source of the information by displaying the text of the assignment attached byto the licensor and, as far as possible, the address where the information waspublished by it._
+* url: _https://geoportal.ancpi.ro/geoportal/catalog/download/download.page_
+* license: _https://geoportal.ancpi.ro/portal/sharing/rest/content/items/8077dd16e957477d9cc9b457f5f17cac/data_
```

Comment:
Do we need to add a `license_en` property? You're quoting text linked in Romanian, but I think there's an official English translation of it, too?

Reply:
Hmm.. I'm still poking around, but do not see an official English version of a license. I have no strong feelings one way or the other about a new `license_text_en` property. If we do add it, though, it should be an optional property.

Reply:
I added the `license_text_en` property in 391394d.. I also updated the main README file and source template to include a note about the new property.

The `license_text` is now in Romanian, a translated version in `license_text_en`. I think this can be merged now, unless there is another English version that should be used.

Reply:
Would the WOF-ism be license_text_eng (3 char)?

Reply:
updated in 8ef1ad6